README.md

```diff
@@ -382,16 +382,11 @@ If you are using Create React Native App / Expo, no Android specific installatio
 
 - For **Expo**, there are at least 2 solutions to fix it:
 
-  - Wrap GiftedChat in a [`KeyboardAvoidingView`](https://facebook.github.io/react-native/docs/keyboardavoidingview). This should only be done for Android, as `KeyboardAvoidingView` may conflict with the iOS keyboard avoidance already built into GiftedChat, e.g.:
+  - Add a [`KeyboardAvoidingView`](https://facebook.github.io/react-native/docs/keyboardavoidingview) below GiftedChat, wrapped in a parent whose `flex` is 1. This should only be done for Android, as `KeyboardAvoidingView` may conflict with the iOS keyboard avoidance already built into GiftedChat, e.g.:
 ```
 <View style={{ flex: 1 }}>
-   {
-      Platform.OS === 'android' ?
-         <KeyboardAvoidingView behavior="padding">
-            <GiftedChat />
-         </KeyboardAvoidingView> :
-         <GiftedChat />
-   }
+   <GiftedChat />
+   {Platform.OS === 'android' ? <KeyboardAvoidingView behavior="padding" />}
```

Comment:
I think you're missing the expression to return if your condition is false in your ternary operator.

Something like below would work instead?

   {Platform.OS === 'android' ? <KeyboardAvoidingView behavior="padding" /> : null}

Reply:
d'oh! Thanks.